packages/cli/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@lerna/cli",
+  "version": "2.5.1",
+  "description": "Tool for managing JavaScript projects with multiple packages",
+  "bin": {
+    "lerna": "cli.js"
+  },
+  "files": [
+    "cli.js"
+  ],
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lerna/lerna.git"
+  },
+  "author": "Sebastian McKenzie <sebmck@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/lerna/lerna/issues"
+  },
+  "homepage": "https://lernajs.io/",
+  "dependencies": {
+    "@lerna/core": "^2.5.1",
+    "import-local": "^0.1.1",
+    "npmlog": "^4.1.2"
+  },
+  "devDependencies": {
+    "@lerna/test": "^2.5.1",
+    "jest": "^19.0.2"
```

Comment:
All of the jest stuff (config, deps,  and running) should be done from the root.

Reply:
Understood. I tried to keep the modules completely self contained regarding any executable code, including tests. Is that no design goal for you?

Reply:
No, I'm a big believer in hoisting all testing and building to the root of a lerna repo, per-package stuff is a recipe for increased maintenance costs for little (if any) gain.

Reply:
Ok, I'll change this accordingly